actors/multisig/src/lib.rs

```diff
@@ -257,7 +257,7 @@ impl Actor {
     pub fn add_signer(rt: &mut impl Runtime, params: AddSignerParams) -> Result<(), ActorError> {
         let receiver = rt.message().receiver();
         rt.validate_immediate_caller_is(std::iter::once(&receiver))?;
-        let resolved_new_signer = resolve_to_actor_id(rt, &params.signer)?;
+        let resolved_new_signer = resolve_to_actor_id(rt, &params.signer, false)?;
```

Comment:
`true`

Reply:
@anorth for adding a new signer, do we need to check for existence ?
as the signer will not exist resolve_to_actor_id will fail

Reply:
Yes. If the address resolves to an ID (because it's already an ID address) but the ID does not exist, this is almost certainly not what the caller wanted. They won't control the signer.